Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/DocumentAdapter.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/DocumentPropertyEditor/DocumentAdapter.h>
+
+namespace AZ::DocumentPropertyEditor
+{
+    void DocumentAdapter::ConnectResetHandler(ResetEvent::Handler& handler)
+    {
+        handler.Connect(m_resetEvent);
+    }
+
+    void DocumentAdapter::ConnectChangedHandler(ChangedEvent::Handler& handler)
+    {
+        handler.Connect(m_changedEvent);
+    }
+
+    void DocumentAdapter::ResetDocument()
```

Comment:
Is this function expected to do a reset of the dom in the future? Or is this merely a notification event? If so, changing this to NotifyDocumentReset() might be a more accurate name for this.

Reply:
It's a notification event, I'll go with `NotifyResetDocument`